.rubocop.yml

```diff
@@ -0,0 +1,4 @@
+inherit_gem:
+  govuk-lint: "configs/rubocop/all.yml"
```

Comment:
Should all of the Rails specific cops be disabled?

Reply:
I'm not sure if they run / interfere with anything?

Reply:
They're not failing anything at the moment, if it's a problem it can always be disabled later